scripts/go.mod

```diff
@@ -1,7 +1,8 @@
 module _ // go.mod created for tooling dependencies
 
-go 1.22
-toolchain go1.24.1
+go 1.23.7
```

Comment:
should be `go 1.24.2`?

Reply:
https://go.dev/doc/modules/gomod-ref#go

`The go line must be greater than or equal to the go line of all dependencies.`

IIUC the Go toolchain selected 1.23 because it is the min required version from all deps.